File: Training/TOKI/Kelas Pembelajaran Pemrograman/Pemrograman Kompetitif Dasar/Chapter 1 - Perkenalan Pemrograman Kompetitif/hapusSatuSaja.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    string input1, input2;

    cin >> input1;
    cin >> input2;

    if (input2.length() > input1.length())
        input2.swap(input1);

    if (input2.length() + 1 == input1.length())
    {
        int i = 0;

        while ((input1[i] == input2[i]) && (i < input2.length()))
            i++;

        if (i == input2.length())
        {
            cout << "Tentu saja bisa!\n";
        }
        else
        {
            string end1 = input1.substr(i + 1, input1.length() - (i + 1));
            string end2 = input2.substr(i, input2.length() - i);

            if (end1 == end2)
                cout << "Tentu saja bisa!\n";
            else
                cout << "Wah, tidak bisa :(\n";
        }
    }
    else
    {
        cout << "Wah, tidak bisa :(\n";
    }
}

File: Training/TOKI/Kelas Pembelajaran Pemrograman/Pemrograman Kompetitif Dasar/Chapter 1 - Perkenalan Pemrograman Kompetitif/hapusSatuSaja.py
string1 = input()
string2 = input()

if (len(string2) > len(string1)):
    temp = string1
    string1 = string2
    string2 = temp

if (len(string2) + 1 == len(string1)):
    i = 0
    while ((string1[i] == string2[i]) and (i < len(string2))):
        i += 1

    if (i == len(string1) - 1):
        print("Tentu saja bisa!")
    else:
        end1 = string1[i+1:]
        end2 = string2[i:]

        if (end1 == end2):
            print("Tentu saja bisa!")
        else:
            print("Wah, tidak bisa :(")
else:
    print("Wah, tidak bisa :(")